indexer/centers_table.cpp

```diff
@@ -315,15 +315,15 @@ void CentersTableBuilder::Freeze(Writer & writer) const
     for (auto const & offset : offsets)
       builder.push_back(offset);
 
-    header.m_positionsOffset = writer.Pos() - startOffset;
+    header.m_positionsOffset = static_cast<uint32_t>(writer.Pos() - startOffset);
```

Comment:
Could you please CHECK here that writer.Pos() - startOffset fits into uint32_t? This code is used during maps generation, so CHECK is ok here.